requirements-dev.txt

```diff
@@ -42,6 +42,8 @@ filelock==3.11.0
     #   virtualenv
 h11==0.14.0
     # via uvicorn
+httpx==0.24.1
+    # via datahub (pyproject.toml)
```

Comment:
I get a few more changes to this file when I run `pip-compile --extra dev -o requirements-dev.txt`. Can you try doing that again?